### ISSUE TITLE
How to conjugate in all persons?

### ISSUE THREAD

**Issue by scarroll32** [SekouD/mlconjug#83]
How do we conjugate in all persons, eg:

```
je capture
tu captures
il;elle;on capture
nous capturons
vous capturez
ils;elles capturent
```

In French http://verbix.com/webverbix/go.php?T1=capturer&Submit=Go&D1=3&H1=103

**Comment by SekouDiaoNlp** [comment]
Hi @seanfcarroll , I am @SekouDiaoNlp, formerly on GitHub as @SekouD and I am the original author of mlconjug. I created a new GitHub account at @SekouDiaoNlp as my former account was mostly for my academic projects, while my new account @SekouDiaoNlp is more focused on my professional projects as well as my open-source projects.
I have just released a new version of mlconjug called mlconjug3, to indicate that I dropped support for Python 2.x as it has been deprecated by the end of 2019.

To install it just run :
pip install mlconjug3


And to answer your initial question, if you want to have the pronouns of a given verb conjugation you can check the documentation at https://mlconjug3.readthedocs.io/en/latest/modules.html#mlconjug.mlconjug.Conjugator.conjugate

Basically the method conjugate(verb, subject='abbrev') is the default behavior of the method.

If you want to have all pronouns explicitly listed you should call the method conjugate this way :

conjugate(verb, subject='pronoun')

Have a nice day and thank you for using mlconjug.

Cheers!
